### PR TITLE
Bugfix: bowser bottom bar navigation

### PR DESCRIPTION
**Pull request by estebanmino**
**Description**

This PR fixes three bugs I found investigating https://www.fabric.io/metamask/android/apps/io.metamask/issues/5c9561a5f8b88c296331b222/sessions/5D44847E034400015FF94C2A629752E5_DNE_0_v2?build=109182544

- Minor bug with goForward button styles, was enabled even though it wasn't possible.
- `double java.lang.Double.doubleValue()' on a null object reference` crash. Going back and forward very quickly in the browser makes the app crash, in this PR `webview` ref is always defined but when forcing reload `{ current }` will be null, avoiding calling to an unmounted webview ref.
- `double java.lang.Double.doubleValue()' on a null object reference` crash. Reproduction steps: Go to uniswap then go back before connect modal prompts, you'll get to homescreen with a connect modal, if you press cancel the app will crash. This not always happens, but at least one of three times it does. This PR makes `approvalRequest` callbacks object undefined if the webview was reloaded, avoiding calling an undefined object with the user response (reject or connect).

**Checklist**

* [ ] There is a related GitHub issue
* [ ] Tests are included if applicable
* [ ] Any added code is fully documented

**Issue**

Resolves #???

### CODE REVIEW

File: app/components/UI/BrowserBottomBar/index.js [hunk 1]
```diff
@@ -102,14 +102,9 @@ export default class BrowserBottomBar extends PureComponent {
...
-					<Icon
-						name="angle-left"
-						disabled={!canGoBack}
-						size={24}
-						style={[styles.icon, !canGoBack ? styles.disabledIcon : {}]}
-					/>
+					<Icon name="angle-left" size={24} style={[styles.icon, !canGoBack ? styles.disabledIcon : {}]} />
 				</TouchableOpacity>
-				<TouchableOpacity onPress={goForward} style={styles.iconButton}>
+				<TouchableOpacity onPress={goForward} style={styles.iconButton} disabled={!canGoForward}>
```
Comment (brunobar79) [inline app/components/UI/BrowserBottomBar/index.js]: 👏

File: app/components/Views/BrowserTab/index.js [hunk 2]
```diff
@@ -1012,9 +1012,13 @@ export class BrowserTab extends PureComponent {
 			if (!isInitialReload) {
 				this.isReloading = true;
 			}
+			// As we're reloading to other url we should remove this callback
+			this.approvalRequest = undefined;
 			const url2Reload = this.state.inputValue;
 			// Force unmount the webview to avoid caching problems
 			this.setState({ forceReload: true }, () => {
+				// Make sure we're not calling last mounted webview during this time threshold
+				this.webview.current = null;
```
Comment (brunobar79) [inline app/components/Views/BrowserTab/index.js]: This line is a bit scary to me but I understand the reason behind it, so if it passes QA I'm good with it.

**Review by brunobar79 — APPROVED** [review]
LGTM!

**Comment by ibrahimtaveras00** [comment]
Fix looks good on android, and I also sanity checked iOS; QA Passed 👍

**Pull request merged**